### PR TITLE
PSMDB-1077 Release notes 5.0.9-8

### PR DESCRIPTION
**Pull request by nastena1606**
deleted:    source/conf-netlify/conf.py
	modified:   source/conf.py
	new file:   source/release_notes/5.0.9-8.rst
	modified:   source/release_notes/index.rst

### CODE REVIEW

File: source/release_notes/index.rst [hunk 1]
```diff
@@ -7,6 +7,7 @@ Percona Server for MongoDB |version| Release Notes
 .. toctree::
    :maxdepth: 1
 
+   5.0.9-8
```
Comment (igorsol) [inline source/release_notes/index.rst:10]: it seems suspicious that new line is not similar to the previous ones. Is this what was intended?

File: source/release_notes/5.0.9-8.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Release Highlights
+==================
+
+* Support of :ref:`multiple KMIP servers <kmip>` adds failover to your data-at-rest encryption setup.
+* The support of encrypted KMIP client certificates and keys through the password setting. 
+
+The bug fixes, provided by MongoDB and included in Percona Server for MongoDB, are the following:
+
+* `SERVER-65636 <https://jira.mongodb.org/browse/SERVER-65636>`_ - Fixed the limits of LDAP connections per host from total number of connections to the number of in-progress connections.
```
Comment (igorsol) [inline source/release_notes/5.0.9-8.rst]: this is enterprise-only bug fix so it has nothing to do with PSMDB

File: source/release_notes/5.0.9-8.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+* `SERVER-64822 <https://jira.mongodb.org/browse/SERVER-64822>`_ - Fixed the issue with releasing the critical state too early when sharding an empty collection. This could result in unwanted writes to that collection.
+* `WT-9096 <https://jira.mongodb.org/browse/WT-9096>`_ - Fixed the wrong key/value returning during search near when the key doesn't exist
+  
+Find the full list of changes in the `MongoDB 5.0.9 Community Edition release notes <https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.9---may-31--2022>`_.
+
+Improvements
+================================================================================
+
+* :jirabug:`PSMDB-1045`: Add support for several KMIP servers 
+* :jirabug:`PSMDB-1043`: The ``kmipClientCertificateFile`` option now includes both the client private key and public certificate
```
Comment (igorsol) [inline source/release_notes/5.0.9-8.rst]: The file specified by kmipClientCertificateFile now includes both the client private key and public certificate. kmipClientKeyFile removed.

**Review by igorsol — CHANGES_REQUESTED** [review]

File: source/release_notes/index.rst (hunk 1 above)
Comment (nastena1606) [reply source/release_notes/index.rst:10]: Yes, all this fuss is to have release dates included in the index. With 5.0.9-8, I've added the release date to the heading so it automatically appears in the index.

File: source/release_notes/5.0.9-8.rst (hunk 2 above)
Comment (nastena1606) [reply source/release_notes/5.0.9-8.rst]: Removed

File: source/release_notes/5.0.9-8.rst (hunk 3 above)
Comment (nastena1606) [reply source/release_notes/5.0.9-8.rst]: Amended

File: source/release_notes/5.0.9-8.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+* `SERVER-64822 <https://jira.mongodb.org/browse/SERVER-64822>`_ - Fixed the issue with releasing the critical state too early when sharding an empty collection. This could result in unwanted writes to that collection.
+* `WT-9096 <https://jira.mongodb.org/browse/WT-9096>`_ - Fixed the wrong key/value returning during search near when the key doesn't exist
+  
+Find the full list of changes in the `MongoDB 5.0.9 Community Edition release notes <https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.9---may-31--2022>`_.
+
+Improvements
+================================================================================
+
+* :jirabug:`PSMDB-1045`: Add support for several KMIP servers 
+* :jirabug:`PSMDB-1043`: Remove the ``kmipClientKeyFile``option and include both the client private key and public certificate in the file specified by the``kmipClientCertificateFile`` option.
```
Comment (igorsol) [inline source/release_notes/5.0.9-8.rst]: Missing some spaces around `` marks

**Review by igorsol — APPROVED** [review]

File: source/release_notes/5.0.9-8.rst [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+Percona Server for MongoDB 5.0.9-8 is an enhanced, source available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 5.0.9 Community Edition.
+It supports MongoDB 5.0.9 protocols and drivers.
+
+Release Highlights
+==================
+
+* Support of :ref:`multiple KMIP servers <kmip>` adds failover to your data-at-rest encryption setup.
+* The support of encrypted KMIP client certificates and keys through the password setting.
```
Comment (spron-in) [inline source/release_notes/5.0.9-8.rst]: Can we expand the value a bit here? So I can now set the password for KMIP certs. What does it give me, as a user? Why should this excite me to update?

File: source/release_notes/5.0.9-8.rst [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+:Installation: :ref:`install`
+
+Percona Server for MongoDB 5.0.9-8 is an enhanced, source available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 5.0.9 Community Edition.
+It supports MongoDB 5.0.9 protocols and drivers.
+
+Release Highlights
+==================
+
+* Support of :ref:`multiple KMIP servers <kmip>` adds failover to your data-at-rest encryption setup.
```
Comment (spron-in) [inline source/release_notes/5.0.9-8.rst:17]: Love this one. It is extremely concise, explains what is it about and communicates the value.

**Review by spron-in — CHANGES_REQUESTED** [review]

**Comment by kwilktx** [comment]
Add this improvement to the Release Highlights section

"Improvements to initial syncs from a secondary sync source"

This comes via SERVER-66089 but we do not need to include that information

**Comment by kwilktx** [comment]
- relates to balancing of shards 
Add this to the Bugs Fixed section:

"[SERVER-66041](https://jira.mongodb.org/browse/SERVER-66041)  - Prevent shard imbalances due to chunk cloner disregarding oversized chunks that contain only 1 oversized document"

**Review by kwilktx — APPROVED** [review]
Looks good after my 2 comments added.  Thanks!

File: source/release_notes/5.0.9-8.rst (hunk 5 above)
Comment (nastena1606) [reply source/release_notes/5.0.9-8.rst]: Amended

**Pull request merged**